packages/ui/src/components/vuestic-components/va-select/VaSelect.vue

```diff
@@ -542,6 +578,11 @@ export default class VaSelect extends mixins(
       this.hintedSearchQuery += event.key
     }
 
+    if (this.doShowSearchInput) {
```

Comment:
quite a strange naming imho
maybe should better be "showSearchInput"?

Reply:
Sounds good.